### PR TITLE
Update subscription link radiolab-kids.vue

### PR DESCRIPTION
**Pull request by kareemlawrence**
Replace Chartable link with link.to

### CODE REVIEW

File: pages/radiolab-kids.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@ const apiUrl = `${ config.API_URL }/api/v3/channel/shows/radiolab-kids/podcasts/
             <h1 class="mb-4 h2 font-normal">Radiolab for Kids</h1>
             <p class>
               Welcome, nature lovers, to the home of the Terrestrials podcast and family-friendly Radiolab episodes about nature. Every other week, host Lulu Miller will take you on a nature walk to encounter a plant or animal behaving in ways that will surprise you. Squirrels that can regrow their brains, octopuses that can outsmart their human captors, honeybees that can predict the future. You don't have to be a kid to listen, just someone who likes to see the world anew.<br>
-              <a href="https://link.chtbl.com/radiolabforkids?sid=rlkidswebsite" target="_blank" rel="noopener noreferrer">Subscribe now!</a> 
+              <a href="https://lnk.to/radiolabforkids!rlkidswebsite" target="_blank" rel="noopener noreferrer">Subscribe now!</a>
```
Comment (kedmundson) [inline pages/radiolab-kids.vue:35]: Did this string `?sid=` get removed by accident?

Comment (kareemlawrence) [reply pages/radiolab-kids.vue:35]: Yes.

Comment (kareemlawrence) [reply pages/radiolab-kids.vue:35]: Actually no. Looking back on the ticket it doesn't have that sid.

Comment (kedmundson) [reply pages/radiolab-kids.vue:35]: That's not typically a valid URL. Could I take a look at the ticket?

Comment (kareemlawrence) [reply pages/radiolab-kids.vue:35]: https://nypublicradio-digital.atlassian.net/browse/BE-47

**Review by kedmundson — APPROVED** [review]

**Pull request merged**